### ISSUE TITLE
Uncaught SyntaxError: unexpected token: ':'

### ISSUE THREAD

**Issue by apzuk3** [ZEGOCLOUD/zego_uikit_prebuilt_web#2]
Hi! 
We are trying to integrate ZEGO prebuilt videocall one-on-one screen into our Angular application but unfortunately getting the following error 

```
Uncaught SyntaxError: Unexpected token ':'
    at ./node_modules/flv.js/dist/flv.js (zego-uikit-prebuilt.js:1258:1)
    at __nested_webpack_require_6011602__ (zego-uikit-prebuilt.js:4766:42)
    at eval (video.tsx:6:64)
    at ./src/sdk/view/components/zegoMedia/video.tsx (zego-uikit-prebuilt.js:2182:1)
    at __nested_webpack_require_6011602__ (zego-uikit-prebuilt.js:4766:42)
    at eval (zegoVideoPlayer.tsx:11:85)
    at ./src/sdk/view/pages/ZegoRoom/components/zegoVideoPlayer.tsx (zego-uikit-prebuilt.js:2446:1)
    at __nested_webpack_require_6011602__ (zego-uikit-prebuilt.js:4766:42)
    at eval (zegoOne2One.tsx:9:74)
    at ./src/sdk/view/pages/ZegoRoom/components/zegoOne2One.tsx (zego-uikit-prebuilt.js:2380:1)
```

The component 

```
import { Component, ElementRef, ViewChild, AfterViewInit } from '@angular/core';
import { ZegoUIKitPrebuilt } from '@zegocloud/zego-uikit-prebuilt';

// get token
function generateToken(tokenServerUrl: string, userID: string) {
  // Obtain the token interface provided by the App Server
  return fetch(
    `${tokenServerUrl}/access_token?userID=${userID}&expired_ts=7200`,
    {
      method: 'GET',
    }
  ).then((res) => res.json());
}

function randomID(len: number) {
  let result = '';
  if (result) return result;
  const chars =
      '12345qwertyuiopasdfgh67890jklmnbvcxzMNBVCZXASDQWERTYHGFUIOLKJP',
    maxPos = chars.length;
  len = len || 5;
  for (let i = 0; i < len; i++) {
    result += chars.charAt(Math.floor(Math.random() * maxPos));
  }
  return result;
}

export function getUrlParams(url = window.location.href) {
  const urlStr = url.split('?')[1];
  return new URLSearchParams(urlStr);
}
@Component({
  selector: 'app-connect',
  templateUrl: './connect.component.html',
  styleUrls: ['./connect.component.scss'],
})
export class ConnectComponent implements AfterViewInit {
  @ViewChild('room')
  root!: ElementRef;

  ngAfterViewInit() {
    console.log(ZegoUIKitPrebuilt.Host);

  //   const roomID = getUrlParams().get('roomID') || randomID(5);
  //   const userID = randomID(5);
  //   const userName = randomID(5);
  //   // generate token
  //   generateToken('https://nextjs-token.vercel.app/api', userID).then((res) => {
  //     const token = ZegoUIKitPrebuilt.generateKitTokenForProduction(
  //       1484647939,
  //       res.token,
  //       roomID,
  //       userID,
  //       userName
  //     );

  //     console.log(token);
  //     // create instance object from token
  //     const zp = ZegoUIKitPrebuilt.create(token);

  //     console.log(
  //       'this.root.nativeElement',
  //       this.root.nativeElement.clientWidth
  //     );
  //     // start the call
  //     zp.joinRoom({
  //       container: this.root.nativeElement,
  //       sharedLinks: [
  //         {
  //           name: 'Personal link',
  //           url:
  //             window.location.origin +
  //             window.location.pathname +
  //             '?roomID=' +
  //             roomID,
  //         },
  //       ],
  //       scenario: {
  //         mode: ZegoUIKitPrebuilt.GroupCall, // To implement 1-on-1 calls, modify the parameter here to [ZegoUIKitPrebuilt.OneONoneCall].
  //       },
  //     });
  //   });
  }
}
```
So even if I just try to access `ZegoUIKitPrebuilt.Host` it throws the error above

`package.json`

```
{
  "name": "name",
  "version": "0.1.3",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "watch": "ng build --watch --configuration development",
    "test": "ng test",
    "lint": "ng lint --quiet"
  },
  "private": true,
  "dependencies": {
    "@angular-slider/ngx-slider": "^2.0.4",
    "@angular/animations": "^15.0.0",
    "@angular/cdk": "^15.0.2",
    "@angular/common": "^15.0.0",
    "@angular/compiler": "^15.0.0",
    "@angular/core": "^15.0.0",
    "@angular/fire": "^7.0.4",
    "@angular/forms": "^15.0.0",
    "@angular/material": "^15.0.2",
    "@angular/platform-browser": "^15.0.0",
    "@angular/platform-browser-dynamic": "^15.0.0",
    "@angular/router": "^15.0.0",
    "@fortawesome/angular-fontawesome": "^0.12.0",
    "@fortawesome/fontawesome-free": "^6.0.0",
    "@fortawesome/fontawesome-svg-core": "^6.2.0",
    "@fortawesome/free-brands-svg-icons": "^6.2.1",
    "@fortawesome/free-solid-svg-icons": "^6.2.0",
    "@ng-bootstrap/ng-bootstrap": "^14.0.0",
    "@ngneat/input-mask": "^6.0.0",
    "@popperjs/core": "^2.11.6",
    "@zegocloud/zego-uikit-prebuilt": "^1.6.2",
    "autocompleter": "^7.1.0",
    "bootstrap": "^5.2.3",
    "crypto-browserify": "^3.12.0",
    "crypto-js": "^4.1.1",
    "date-fns": "^2.29.1",
    "inputmask": "^5.0.7",
    "ngx-drag-to-select": "^5.0.1",
    "ngx-mask": "^15.0.2",
    "ngx-sharebuttons": "^11.0.0",
    "rxjs": "~7.5.0",
    "timezones-list": "^3.0.1",
    "tslib": "^2.4.0",
    "typeahead": "^0.2.2",
    "zone.js": "~0.12.0"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^15.0.3",
    "@angular-eslint/builder": "15.2.0",
    "@angular-eslint/eslint-plugin": "15.2.0",
    "@angular-eslint/eslint-plugin-template": "15.2.0",
    "@angular-eslint/schematics": "15.2.0",
    "@angular-eslint/template-parser": "15.2.0",
    "@angular/cli": "~15.0.3",
    "@angular/compiler-cli": "^15.0.0",
    "@angular/localize": "^15.0.0",
    "@types/calendar": "^0.1.2",
    "@types/crypto-js": "^4.1.1",
    "@types/inputmask": "5.0.0",
    "@types/jasmine": "~4.3.0",
    "@types/jest": "^29.2.4",
    "@types/mocha": "^10.0.1",
    "@typescript-eslint/eslint-plugin": "5.48.1",
    "@typescript-eslint/parser": "5.48.1",
    "autoprefixer": "^10.4.13",
    "eslint": "^8.31.0",
    "jasmine-core": "~4.5.0",
    "karma": "~6.4.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.2.0",
    "karma-jasmine": "~5.1.0",
    "karma-jasmine-html-reporter": "~2.0.0",
    "postcss": "^8.4.20",
    "typescript": "~4.8.2"
  }
}
``` 

Anyone able to give some help? 
Thanks!

**Comment by apzuk3** [comment]
<img width="1188" alt="Screenshot 2023-01-31 at 13 46 56" src="https://user-images.githubusercontent.com/3325376/215676395-4921ac26-1d0a-4adf-86bb-0e0b01d73455.png">

I have just noticed this, but can not figure out why this is happening 🤔

**Comment by apzuk3** [comment]
Upd: this problem seem to exists only with `ng serve` mode. When built and deployed to prod it works fine.

**Comment by apzuk3** [comment]
I just disabled `sourceMap` for development so we can continue, but it would be nice to enable it back. I will leave this here if anyone has any idea how to solve it

**Comment by listen936** [comment]
This issue was resolved in 1.8.9
